### PR TITLE
Feature two

### PR DESCRIPTION
**Pull request by paulzay**
### What does this pr do? 
Build an eventbrite-like app
### Description of tasks to be completed
1. Create signup and sign in feature
2. Create events features
3. CReate invites to events

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Past Events:<P>
+
+<% Event.past.each do |e| %>
+	<%= link_to e.name, e %>, created by <%= e.creator.name%> <p>
+<% end %>
+<p>Upcoming Events:<p>
+
+<% Event.upcoming.each do |e| %>
+	<%= link_to e.name, e %>, created by <%= e.creator.name%> <p>
+<% end %>
```
Comment (trekab) [inline app/views/events/index.html.erb:14]: Please remove this (`Events.past` and `Events.upcoimng`) logic from the view and create:
- [ ] `@past_events = Event.past` to the Events' controller index action.
- [ ] `@upcoming_events = Event.upcoming` to the Events' controller index action.

Then you can conveniently rely on `@past_events` and `@upcoming_events` in your view. The is the cleaner way (Check out [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make#common-mistake-2-putting-too-much-logic-in-the-view) about putting too much logic in the view)

Comment (trekab) [inline app/views/events/index.html.erb]: Please remove this (`Events.past` and `Events.upcoimng`) logic from the view and create:

 - [ ] `@past_events = Event.past` to the Events' controller index action.
 - [ ] `@upcoming_events = Event.upcoming` to the Events' controller index action.

Then you can conveniently rely on `@past_events` and `@upcoming_events` in your view. This is the cleaner way (Check out [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make#common-mistake-2-putting-too-much-logic-in-the-view) about putting too much logic in the view)

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
+  get '/signup', to: 'users#new'
+
+  resources :events
+  get '/new', to: 'events#new'
+  post '/create', to: 'events#create'
+  delete '/delete', to: 'events#destroy'
+  put '/edit', to: 'events#update'
```
Comment (trekab) [inline config/routes.rb]: This isn't necessary once you have used `resources`. It is currently blocking logged-in users from creating new events.

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @paulzay and @DickensJuma! You have done well on this project. There are just a few changes that require your attention before you can have it complete. These include:

- [ ] Check the comments below in the code for the specifics.
- [ ] __[OPTIONAL]__ Consider using routes only for used actions, e.g. `resources :events, only: [:new, :create, :show, :index]`

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved :heavy_check_mark:
#### Hi @paulzay and @DickensJuma :clap: :clap:
#### Well done team. You have done a good job :muscle: This project is approved :heavy_check_mark:

#### _Reviewed by Temitayo Adeleke_

**Pull request merged**